modules/reprezen-openapi-gentemplates/pom.xml

```diff
@@ -36,10 +36,5 @@
             <groupId>com.reprezen.genflow</groupId>
             <artifactId>swagger-ui</artifactId>
         </dependency>
-        <dependency>
-            <groupId>com.konghq</groupId>
-            <artifactId>unirest-java</artifactId>
-            <classifier>standalone</classifier>
-        </dependency>
```

Comment:
As I understood it, `modules/reprezen-openapi-gentemplates/pom.xml ` has dependencies specific to the OpenAPI GenTemplates provided by RepreZen.  `modules/genflow-api-openapi/pom.xml ` has dependencies that apply to _all_ OpenAPI GenTemplates from _all_ providers. Is that right? If so, wouldn't it work to add these dependencies to `modules/reprezen-openapi-gentemplates/pom.xml ` , so they are specific to RepreZen GenTemplates?

Reply:
`reprezen-openapi-gentemplates` is the group that contains all our gentemplates. `genflow-api-openapi` contains the actual classes to create OpenAPI gentemplates.